helm/values-dev.yaml

```diff
@@ -84,11 +85,14 @@ microservice-chart:
     MONGO_SSL_ENABLED: "true"
     ECOMMERCE_STORAGE_QUEUE_ACCOUNT_NAME: pagopadweuecommercesa
     ECOMMERCE_STORAGE_QUEUE_ENDPOINT: https://pagopadweuecommercesa.queue.core.windows.net/pagopadweuecommercesa
+    REFUND_RETRY_EVENT_OFFSET_SECONDS: 600
```

Comment:
🚲: OFFSET -> INTERVAL / BASE_INTERVAL / INITIAL_INTERVAL

Reply:
I like BASE_INTERVAL just because this is the base from which derive the effective visibility timeout of the event

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-scheduler-service/pull/34/commits/4e7ae58b849300572ed71d7b0df8569500cc53c6
Contextually open PR on local for system property refactoring
https://github.com/pagopa/pagopa-ecommerce-local/pull/48